Description: Correct pod NAME issue in LedgerSMB/DBObject/TaxForm.pm
 Has the module name but does not have the dash plus short description in the
 NAME entry.
Bug: http://sourceforge.net/tracker/?func=detail&aid=3472573&group_id=175965&atid=1005191
Author: Robert James Clay <jame@rocasa.us>

--- a/LedgerSMB/DBObject/TaxForm.pm
+++ b/LedgerSMB/DBObject/TaxForm.pm
@@ -2,7 +2,7 @@
 
 =head1 NAME
 
-LedgerSMB:DBObject::TaxForm
+LedgerSMB:DBObject::TaxForm - Includes methods for saving and retrieving tax forms.
 
 =head1 SYNOPSIS
 
